app/script/view_model/PanelViewModel.js

```diff
@@ -226,14 +244,18 @@ z.viewModel.PanelViewModel = class PanelViewModel {
         break;
     }
 
-    this.previousState(this.state());
+    this.previousState(state);
+    this.exitingState(undefined);
 
-    const panelStateElementId = this._getElementIdOfPanel(this.state());
-    $(`#${panelStateElementId}`).removeClass('panel__page--visible');
+    const panelStateElementId = this._getElementIdOfPanel(state);
+    $(`#${panelStateElementId}`).removeClass(
+      'panel__page--visible panel__page--move-out--left panel__page--move-out--right'
```

Comment:
let's move this to a variable if prettier doesn't like to put it into a single line